### ISSUE TITLE
questions about the input_height and input_width

### ISSUE THREAD

**Issue by lzzlxxlsz** [LeeDoYup/AnoGAN#26]
last time,I use my data with the size of 1544*1284,and I resized it with 178*218 as the  celeba dataset,but the result is not good.The generated picture is not clear,so I want to ask how to choose the size of data

**Comment by LeeDoYup** [comment]
When you train the GAN with your own data, hyperparameter tuning is needed.
(because GAN is very sensitive to dataset and hyperparameters)

**Comment by lzzlxxlsz** [comment]
Thanks,that is to say,I don't need to resize my dataset,but when I set the input_height and width with a big size,it dosen't work.Another question is that the picture would lost some information after crop in the training test ,if I want the GAN learn the whole picture ,do I need crop?

**Comment by lzzlxxlsz** [comment]
Sorry,I wonder the hyperparameter, does it contains the learning rate,beta1 and other? Thanks for you if you could explain more detail,and I am feel confused about what aspects should I adjust to suit my dataset
